### PR TITLE
Fix spack build for RAJA: master requires submodules

### PR DESCRIPTION
**Pull request by wrwilliams**
As per subject; it's a trivial change.

### CODE REVIEW

File: var/spack/repos/builtin/packages/raja/package.py [hunk 1]
```diff
@@ -29,6 +29,6 @@ class Raja(CMakePackage):
     """RAJA Parallel Framework."""
     homepage = "http://software.llnl.gov/RAJA/"
 
-    version('git', git='https://github.com/LLNL/RAJA.git', branch="master")
+    version('git', git='https://github.com/LLNL/RAJA.git', branch="master", submodules="True")
```
Comment (davydden) [inline var/spack/repos/builtin/packages/raja/package.py]: could you please rename it to `develop`? this is a special name which makes sure it's greater than any other version, i.e. `x.y.z`.

Comment (wrwilliams) [reply var/spack/repos/builtin/packages/raja/package.py]: I can if folks like, but the git version is the only version present right now--figured that could wait until RAJA gets proper in-package versioning.

Comment (davydden) [reply var/spack/repos/builtin/packages/raja/package.py]: > figured that could wait until RAJA gets proper in-package versioning.

sure, but there is really no reason not to fix it now, while you modify the package already.

Comment (davydden) [reply var/spack/repos/builtin/packages/raja/package.py]: FYI http://spack.readthedocs.io/en/latest/packaging_guide.html#version-comparison

**Pull request merged**